images_config/ks-robot-master.cfg

```diff
@@ -123,72 +121,44 @@ glib-compile-schemas $settingsDir
 # Turn on Wake-on-LAN
 ethtool -s eth0 wol g
 
-# Enable shutdown and updates for ernestas without password
-echo "" >> /etc/sudoers
-echo "#Custom rules" >> /etc/sudoers
-echo "ernestas ALL = NOPASSWD: /sbin/shutdown" >> /etc/sudoers
-echo "ernestas ALL = NOPASSWD: /usr/bin/apt-get" >> /etc/sudoers
-
-# Install Google Chrome
-wget https://dl.google.com/linux/direct/google-chrome-stable_current_i386.deb
-dpkg -i google-chrome-stable_current_i386.deb
-
-# Delete Chrome install
-rm google-chrome-stable_current_i386.deb
-
-# Clean
-#apt-get -f -y install
-
-#apt-get -y autoremove
-#apt-get clean
-
 # Add your custom post installation script here. It's not necessary, but you can add something depending on your needs. Here's my script:
 ######################POST-INSTALLATION#######################################
+
+########## SETUP GRUB ##################
 %post --interpreter=/bin/bash
 #!/bin/bash
-apt-get update && apt-get upgrade -y
```

Comment:
do not remove this line, you should upgrade the packages before you start with the configuration.